Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean

```diff
@@ -707,6 +707,9 @@ theorem le_rpow_one_div_iff {x y : ℝ≥0∞} {z : ℝ} (hz : 0 < z) : x ≤ y
   rw [rpow_mul, ← one_div, @rpow_le_rpow_iff _ _ (1 / z) (by simp [hz])]
 #align ennreal.le_rpow_one_div_iff ENNReal.le_rpow_one_div_iff
 
+theorem rpow_one_div_lt_iff {x y : ℝ≥0∞} {z : ℝ} (hz : 0 < z) : x ^ (1 / z) < y ↔ x < y ^ z := by
```

Comment:
Please never use `1 / x`. This spelling is deprecated (although not `deprecated`) and we would like no new lemma to use it.
```suggestion
theorem rpow_inv_lt_iff {x y : ℝ≥0∞} {z : ℝ} (hz : 0 < z) : x ^ z⁻¹ < y ↔ x < y ^ z := by
```
Also, when you add such a lemma, please make sure you add all variants, eg here `y < x ^ z⁻¹`, `y ≤ x ^ z⁻¹`, `x ^ z⁻¹ ≤ y`. (I know the variants of the lemma you want to add already exist, this is just general advice).